python/chronos/test/bigdl/chronos/autots/test_tspipeline.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tempfile
+from unittest import TestCase
+import pytest
+import torch
+
+from torch.utils.data import TensorDataset, DataLoader
+from bigdl.chronos.autots import AutoTSEstimator, TSPipeline
+from bigdl.orca.common import init_orca_context, stop_orca_context
+
+
+def train_data_creator(config):
+    return DataLoader(TensorDataset(torch.randn(10000,
+                                                config.get('past_seq_len', 10),
+                                                config.get('input_feature_num', 2)),
+                                    torch.randn(10000,
+                                                config.get('future_seq_len', 2),
+                                                config.get('output_feature_num', 2))),
+                      batch_size=config.get('batch_size', 32), shuffle=True)
+
+def valid_data_creator(config):
+    return DataLoader(TensorDataset(torch.randn(10000,
+                                                config.get('past_seq_len', 10),
+                                                config.get('input_feature_num', 2)),
+                                    torch.randn(10000,
+                                                config.get('future_seq_len', 2),
+                                                config.get('output_feature_num', 2))),
+                      batch_size=config.get('batch_size', 32), shuffle=False)
+
+
+class TestTSPipeline(TestCase):
+
+    def setUp(self) -> None:
+        pass
+
+    def tearDown(self) -> None:
+        pass
+
+    def test_seq2seq_tsppl_seq2seq_support_dataloader(self):
+        tmp_seq2seq_dir = tempfile.TemporaryDirectory()
+        init_orca_context(cores=4, memory="4g")
```

Comment:
I think we need to set `init_ray_on_spark=True`? Can we pass the ut with this unset?